pages/dkp/kommander/2.3/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -25,7 +25,7 @@ This section describes how to upgrade your Kommander Management cluster and all
   Download the Kommander application definitions:
 
   ```bash
-  wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-applications-v2.2.0.tar.gz"
+  wget "https://downloads.d2iq.com/dkp/v2.3.0/kommander-applications-v2.3.0.tar"
```

Comment:
Not sure, but maybe curl "-fsSL https://downloads.d2iq.com/dkp/v2.2.0/kommander-applications-v2.2.0.tar.gz | tar xzf -" will be more convenient?

Reply:
@gflorinskaya  - The 'tar xzf' would unpack the archive, which is not what we want (dkp cli will unpack the archive itself) Also, the point of this change was to remove the .gz so we don't want to add that back. :-)    But I agree that we should be consistent in the docs and use either 'curl' (preferred) or 'wget' anytime we suggest people download something.  Not sure this PR is the right place to change that, tho.

Reply:
I think if we want to change all commands from `wget` to `curl` we could do that. But I agree this should be done in another PR and we should get eng involved just in case there is a reason wget is preferred in some cases... Maybe we can open a ticket for that?

Reply:
I don't believe this bundle will change, aka it will still be `kommander-applications-v2.3.0.tar.gz`. From what I understand, @makkes only intends to change the image bundle

Reply:
So only files with `kommander-image-bundle` in them will change?

Reply:
based on what I see in https://github.com/mesosphere/kommander/issues/1769, all the image bundle files will change but not the chart bundles or git archive.